pkg/controller/pravega/pravega_segmentstore_test.go

```diff
@@ -356,10 +371,34 @@ var _ = Describe("PravegaSegmentstore", func() {
 					_ = pravega.MakeSegmentStoreExternalServices(p)
 					Ω(err).Should(BeNil())
 				})
+			})
+			Context("Create External service with external service type and access type empty", func() {
+				BeforeEach(func() {
+					p.Spec.Pravega.SegmentStoreExternalServiceType = ""
+					p.Spec.ExternalAccess.Type = ""
+					p.Spec.ExternalAccess.DomainName = "example"
+				})
+				It("should create external service", func() {
 
+					_ = pravega.MakeSegmentStoreExternalServices(p)
+					Ω(err).Should(BeNil())
```

Comment:
since ServiceType was reset we can also check that it is set to default value here

Reply:
intent is to just to cover service is getting created